lib/client.js

```diff
@@ -238,8 +238,10 @@ extend(Raven.prototype, {
     kwargs.request = extend({}, this._globalContext.request, domainContext.request, kwargs.request);
     if (Object.keys(kwargs.request).length === 0) {
       var req = extend({}, this._globalContext.req, domainContext.req, kwargs.req);
-      var parseUser = Object.keys(kwargs.user).length === 0 ? this.parseUser : false;
-      kwargs = extend(kwargs, parsers.parseRequest(req, parseUser));
+      if (Object.keys(req).length > 0) {
+        var parseUser = Object.keys(kwargs.user).length === 0 ? this.parseUser : false;
+        kwargs = extend(kwargs, parsers.parseRequest(req, parseUser));
```

Comment:
so that user parsing is actually just to look for a user if there's one *on* the req, so it's kind of a nonexistent case; if there's a `user` property in context/kwargs otherwise, we still make use of it.